src/swapnaWJun20/SumNosUsingASCII.java

```diff
@@ -0,0 +1,22 @@
+package swapnaWJun20;
+
+public class SumNosUsingASCII {
+	int sum = 0;
+
+	int sumUsingASCII(String input) {
+		for (int index = 0; index < input.length(); index++) {
+			for (int innerIndex = 48; innerIndex <= 57; innerIndex++)
+				if (input.charAt(index) <= innerIndex) {
+					sum += Character.getNumericValue(input.charAt(index));
+					break;
```

Comment:
No need of break needed. 
sum += (str.charAt(index) - '0'); You can use this as well.